Kernel/Memory/MemoryManager.cpp

```diff
@@ -198,14 +198,11 @@ bool MemoryManager::is_allowed_to_mmap_to_userspace(PhysicalAddress start_addres
     if (start_address.offset_addition_would_overflow(range.size()))
         return false;
     auto end_address = start_address.offset(range.size());
-    for (auto& current_range : m_reserved_memory_ranges) {
-        if (current_range.start > start_address)
-            continue;
-        if (current_range.start.offset(current_range.length) < end_address)
-            continue;
-        return true;
-    }
-    return false;
+    return AK::any_of(
```

Comment:
What is the motivation for converting loops to `AK::any_of`, other than to appease an arbitrary clang-tidy rule? I don't personally find this better / easier to read.

Reply:
None really,
mainly to appease clang-tidy
I can drop it if you want

Reply:
IMHO we should drop those changes, and disable that rule if this is all it flags.

Reply:
Yeah, let's drop those changes and disable the rule.